dumpling/export/status.go

```diff
@@ -60,7 +69,16 @@ func (d *Dumper) GetStatus() *DumpStatus {
 	ret.FinishedBytes = ReadGauge(d.metrics.finishedSizeGauge)
 	ret.FinishedRows = ReadGauge(d.metrics.finishedRowsGauge)
 	ret.EstimateTotalRows = ReadCounter(d.metrics.estimateTotalRowsCounter)
-	ret.CurrentSpeedBPS = d.speedRecorder.GetSpeed(int64(ret.FinishedBytes))
+	ret.CurrentSpeedBPS = d.speedRecorder.GetSpeed(ret.FinishedBytes)
+	if d.metrics.progressReady.Load() {
```

Comment:
seems forget to set it 😂 so add an integration test about this behaviour

Reply:
added a integration test https://github.com/pingcap/tidb/pull/38625/commits/f3cdc98cf877eacd9cc1cb0409343ee59570aff6